verifiable-api/types/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+use std::collections::HashMap;
+
+use alloy::{
+    consensus::Account as TrieAccount,
+    eips::BlockId,
+    primitives::{Address, Bytes, B256},
+    rpc::types::{EIP1186StorageProof, Log},
+};
+use serde::{Deserialize, Serialize};
+
+use helios_common::network_spec::NetworkSpec;
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct AccountResponse {
+    pub account: TrieAccount,
+    pub code: Bytes,
+    pub account_proof: Vec<Bytes>,
+    pub storage_proof: Vec<EIP1186StorageProof>,
```

Comment:
nit: `storage_proofs`

Reply:
Yeah, ideally it should be `storage_proofs` but I went with `storage_proof` because that is what is used in JSON-RPC -- see [here](https://github.com/alloy-rs/alloy/blob/79964358d42ff4f27d3cb21570dd8026c01d1e75/crates/rpc-types-eth/src/account.rs#L62-L63).

Reply:
Since this is a new api might as well change it to have it make a bit more sense.